### ISSUE TITLE
hard coded connection group text color

### ISSUE THREAD

**Issue by alanrenmsft** [microsoft/azuredatastudio#13138]
currently the text color for connection group is hard coded as white: https://github.com/microsoft/azuredatastudio/blob/7b54abbc96f62cfbbb4c204a0f22ada581994bd5/src/sql/workbench/contrib/dataExplorer/browser/media/connectionViewletPanel.css#L45

connection group color should be defined as a pair, background-color and color and make sure it meets the contrast ratio requirement
